pkg/codegen/docs/gen.go

```diff
@@ -825,13 +826,17 @@ func (mod *modContext) getProperties(properties []*schema.Property, lang string,
 			propLangName = name
 		}
 
+		propID := strings.ToLower(propLangName + "~" + lang)
```

Comment:
I feel like this is an important detail with this separator. Perhaps we can make this a file-level constant and also add a comment about its role?